src/operator/swapaxis-inl.h

```diff
@@ -0,0 +1,197 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file swapaxis-inl.h
+ * \brief
+ * \author Ming Zhang
+*/
+#ifndef MXNET_OPERATOR_SWAPAXIS_INL_H_
+#define MXNET_OPERATOR_SWAPAXIS_INL_H_
+
+#include <dmlc/logging.h>
+#include <dmlc/parameter.h>
+#include <mxnet/operator.h>
+#include <algorithm>
+#include <map>
+#include <vector>
+#include <string>
+#include <utility>
+#include "./operator_common.h"
+
+namespace mxnet {
+namespace op {
+
+struct SwapAxis{
+enum SwapAxisOpInputs {kData};
+enum SwapAxisOpOutputs {kOut};
+};
+
+using namespace mshadow;
```

Comment:
The reason why using namespace is dangerous is because we are doing https://github.com/dmlc/mxnet/tree/master/amalgamation and using namespace in outside can easily leads to name conflictions